### ISSUE TITLE
[Feature suggestion] Ability to make text smaller or bigger

### ISSUE THREAD

**Issue by LarryIsBetter** [vixalien/sticky#17]
Would be a big help and less eye strain.

**Comment by gforcada** [comment]
Yes please!! Double bonus if `Ctrl +` and `Ctrl -` increase and decrease the overall font size on a note 😄

**Comment by vixalien** [comment]
what would make more sense: zoom level being global, or per note?

**Comment by gforcada** [comment]
🤔 as long as it is easy to change it within a note, then globally might be good enough (I'm thinking on a use case of having only one note open at a time, if having several at the same time, then per note makes more sense IMHO)

**Comment by vixalien** [comment]
sory for the late reply, but I did not fully catch what you meant

when you, say press `<Ctrl>+` does that change the font for the current open note, or for all the notes??

maybe we can implement the two settings, where there is a "preferences" window in the all notes window, and users can also increase the size of each note globally...

**Comment by gforcada** [comment]
Not sure if a preference is enough, a sane default should be good enough 🤓 getting that one might be difficult though 😓 

What's the primary use case of `sticky`? To have several notes shown at the same time? How are colors handled? If each note has its own color and can be configured on a per note basis, maybe that already hints that we might actually want to store the zoom level on a per note basis as well.

Then the _only_ preference I would then add is the default font size/zoom level for __new__ notes.

Does that make sense? 🤔

**Comment by vixalien** [comment]
yep that makes sense, but it will be zoom level (or font scale) rather than
absolute font size (say 10 px) for obvious reasons..

then we'll have 2 options:

1. default font scale (will be applied to new notes)
2. each note will be able to have a different font scale

**Comment by VaZark** [comment]
I'm not sure how the text formatting is handled, but adding markdown support might fix most formatting issues/requests

**Comment by vixalien** [comment]
Text formatting is ha
ndled by `Gtk.TextView`, I just added a convenience structure that can save/restore `Gtk.TextTag`'s (think bold, underline etc) inside the TextView.

Markdown will be nice to have, but I [currently do not have the time](https://gitlab.gnome.org/Teams/Circle/-/issues/166#note_1740195) to implement such a huge change.

Do not

**Comment by azure-glades** [comment]
Bumping this request, I too would like this feature. 
Like others mentioned, using `ctrl +` and `ctrl -` would be used to change zoom globally.

This is what I had in mind:
Open a note
Use keymap to increase/decrease zoom level
Close note.
Next time you open any note, it opens as this zoom level.
(Kinda like when you increase zoom for a site in a browser, next time you open the site it is zoomed in)

Please add it 🙏, I would be willing to work on this.

**Comment by azure-glades** [comment]
@vixalien

**Comment by vixalien** [comment]
Hello, all. I'm working on a new version of sticky, and wish it will have this feature!

**Comment by azure-glades** [comment]
Oh cool! How can I contribute?
